src/main/java/bio/terra/janitor/app/controller/UnauthenticatedApiController.java

```diff
@@ -53,6 +58,7 @@
   @Override
   public ResponseEntity<Void> shutdownRequest() {
     try {
+      flightScheduler.shutdown();
```

Comment:
One unrelated thing I notice: The current shutdown is unauthenticatated api.
https://github.com/DataBiosphere/terra-resource-janitor/blob/master/src/main/resources/api/service_openapi.yaml#L43
Can you also update it here, I don't think we want anybody to shut this down..
But you said this is only callable by k8s, that means we have to put it unauthenticatated to bypass the apache proxy? And after that, I am not sure how we do authZ. (would be easier once we know what the request header looks like for calls from k8s)

Reply:
I'm not sure how we'll do this either, but we should be able to use whatever datarepo is doing for their shutdown endpoint. Filed PF-57 and TODO'd it here. Data repo also has their endpoint as unauthenticated.
Ok to leave any changes here until that bug? We should do it before having "production" deployments. We could pick it up sooner if it feels important.